packages/rocketchat-videobridge/client/actionLink.js

```diff
@@ -4,7 +4,7 @@ RocketChat.actionLinks.register('joinJitsiCall', function(/*message, params*/) {
 
 		let room = RocketChat.models.Rooms.findOne({_id: rid});
 		let currentTime = new Date().getTime();
-		let jitsiTimeout = new Date(room.jitsiTimeout).getTime() || currentTime;
+		let jitsiTimeout = new Date((room && room.jitsiTimeout) || currentTime).getTime();
```

Comment:
ah ok.  This is one of those ugly Javascript things :grin:  Usually `( stuff here )` is evaluated and returns a boolean.  All good.  Just tried it.  Apparently if it doesn't return true / false its not treated as a boolean check.  Weird..